manifests/charts/gateway/templates/deployment.yaml

```diff
@@ -32,30 +32,45 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "gateway.serviceAccountName" . }}
-      {{- if not .Values.runAsRoot }}
       securityContext:
-        runAsUser: 1337
-        runAsGroup: 1337
-        runAsNonRoot: true
-        fsGroup: 1337
+      {{- if .Values.securityContext }}
+        {{- toYaml .Values.securityContext | nindent 8 }}
+      {{- else if (semverCompare ">=1.22-0" .Capabilities.KubeVersion.GitVersion) }}
+        # Safe since 1.22: https://github.com/kubernetes/kubernetes/pull/103326
+        sysctls:
+        - name: net.ipv4.ip_unprivileged_port_start
```

Comment:
cc @howardjohn @Kmoneal 
does it matter for ipv6?

Reply:
I am about 60% sure this applies to ipv6 and is poorly named. I did not test it though